docs/resources/client/level-components/capacity-string.md

```diff
@@ -0,0 +1,157 @@
+| 46 | BatchNodeAddText | Text, Blending, zLayer B1 | -4 |
+| 47 | BatchNodeTextBot2 | Text, zLayer B2 | -12 |
+| 48 | BatchNodeAddTextBot2 | Text, Blending, zLayer B2 | -11 |
+| 49 | BatchNodeTextBot3 | Text, zLayer B3 | -19 |
+| 50 | BatchNodeAddTextBot3 | Text, Blending, zLayer B3 | -18 |
+| 51 | BatchNodeTextBot4 | Text, zLayer B4 | -26 |
+| 52 | BatchNodeAddTextBot5 | Text, Blending, zLayer B4 | -25 |
+| 53 | EffectBatchNodeAddTop4 | Animated, Blending, zLayer T4 | 20 |
+
+## Legacy String
+
+There are other types of Capacity Strings from older versions of the game. The total number of Nodes is used to identify them
+
+| Update | Node count |
+|:-------|:-----------|
+| 1.9 | 4 |
+| 2.0 | 16 |
+
+The Capacity String in 2.0 is quite easy to understand as it is the first 16 BatchNodes of 2.1's Capacity String are exactly the same. 1.9 is quite different with certain index's containing mulitple batchNodes
+ 
+> **Below is a table for 1.9's Capacity String.**  
+> Refer to the [Structure](/resources/client/level-components/Capacity-string?id=structure) section for the properties of the batchNodes
+
+| Index | BatchNode(s) |
+|:------|:-------------|
+| 0 | BatchNode |
+| 1 | BatchNodeAdd</br> BatchNodeAddGlow |
+| 2 | BatchNodeBottom</br> BatchNodeBottom2 |
+| 3 | BatchNodeAddBottom</br> BatchNodeAddBottom2</br> BatchNodeAddBottomGlow</br> BatchNodeAddBottom2Glow |
+
+## High Capacity Mode
+
+In the game options, there is a option the player can toggle called `High Capacity Mode`. when enabled, the Capacity for each BatchNode is set to a hardcoded value. There are 3 capacity sizes
+
+| size | Batch Type |
+|:-----|:-----------|
+| 999 | BatchNode |
+| 200 | EffectBatchNode |
+| 100 | BatchNodeText |
+
+If enabled, the Capacity string is disregarded and these values are used instead.
+
+
+
+## Capacity String Generation
+
+The Capacity String is only generated during verification of the level. When playing the level, the game stores the the highest number of GameObjects that the Batch is responsible for inside its textureAtlas as the capacity. After the level is verified, the following function is called
+
+```cpp
+int capacity = BatchNode->getAtlasCapacity();
+if(capacity <= 29) 
+    return 0;
+
+int newCapacity = capacity * 1.1f;
+
+if(newCapacity < 9999) 
+    return newCapacity;
+
+if(newCapacity >= 9999) 
+    return 9999;
+ ```
+
+After the capacity is calculated, it adds the value to the capacity string. Once the entire string is finished, it is then stored in `GJGameLevel->CapacityString_` where it will then be sent to the servers during the upload
+
+<!-- 
+
+enum BatchType {
+    BatchNodePlayer = 1, // Stuff regarding player -> investigate
+    BatchNodeText = 2, // text object
+    EffectBatchNode = 3, // animated objects
+    DefaultBatchNode = 4, // stuff that arent in other batchNodes added here: ie. trigger objects
+    BatchNode = 5 // any gameobject that doesnt fit into the other categories
+};
+
+
+
+
+-->
```

Comment:
should this enum be left in the comments? it seems like it's explained earlier up in the file anyways
```suggestion
```